include/sentry.h

```diff
@@ -35,8 +35,10 @@ extern "C" {
 
 /* SDK Version */
 #ifndef SENTRY_SDK_NAME
-#    ifdef __ANDROID__
+#    if defined(__ANDROID__)
 #        define SENTRY_SDK_NAME "sentry.native.android"
+#    elif defined(_GAMING_XBOX_SCARLETT)
+#        define SENTRY_SDK_NAME "sentry.native.xbox"
```

Comment:
Unrelated to this PR but maybe we could move the SDK name+version+user-agent defines after platform defines and use these to decide what to set.